### DISCUSSION TITLE
Radxa Rock 3C

### DISCUSSION THREAD

**Discussion by ricag101** [Joshua-Riek/ubuntu-rockchip#1141] [General]
Has anyone had any luck with a Build for the Radxa Rock 3C?
I have attempted to use the Radxa Zero 3 image (since its based on the same RK3566 chip and memory) as a basis, and that ended horribly with the CPU overheating almost instantly.
Any guidance on how to either Build a new Build for the Rock 3C or guidance on how to resolve the use of the Zero 3 as a basis, would be great, or any anticipation if a build would be in the scope for the Rock 3C board?

Thanks

**Comment by rushilo** [comment]
The armbian images linked [here](https://www.armbian.com/cf7stdb-entries/rock-3c/) actually work for me. Some features don't (HDMI audio for one) but it'll get you started.

**Comment by ricag101** [comment]
@rushilo Many thanks, but i am rather looking directly for the ubuntu versions, any idea how i would go about getting started with getting the Ubuntu version running in a similar manner?

**Comment by hakotani-o** [comment]
I don't have rock-3c, so I haven't been able to confirm its operation, but I created a build tool. I was able to start it on orangepi-5-plus.
[rock-3c.tar.gz](https://github.com/user-attachments/files/18492053/rock-3c.tar.gz)

**Reply by ricag101** [reply]
@hakotani-o thanks, as a newbie to this, any guidance on how to use this build tool would be greatly appreciated.

**Reply by hakotani-o** [reply]
./run.sh
